app/Resources/views/blog/index.html.twig

```diff
@@ -10,6 +10,13 @@
                     {{ post.title }}
                 </a>
             </h2>
+            {% if 0 < post.tags | length %}
```

Comment:
What about updating the repository to load with tags? To avoid lazy loading
And btw, inform what is lazy loading :)